modules/src/core/ics02_client/client_def.rs

```diff
@@ -494,7 +493,7 @@ impl ClientDef for AnyClient {
         port_id: &PortId,
         channel_id: &ChannelId,
         sequence: Sequence,
-        ack: Acknowledgement,
```

Comment:
I think the continued use of the Acknowledgement type gives a better indication of what is mean.

Reply:
I think the "acknowledgment commitment" should be verified here instead of `Acknowledgement`.

Reply:
> I think the "acknowledgment commitment" should be verified here instead of `Acknowledgement`.

Maybe give a newtype AcknowledgementCommitment

Reply:
I added PacketCommitment and AcknowledgementCommitment